cactusbot/cached/caching.py

```diff
@@ -1,18 +1,32 @@
+import json
+import time
+
+
 class CacheUtils(object):
 
     # You have found MysticalMage's code, turn back now, no good lies ahead
-    def in_cache_followers(user):
-        """ Returns if user is in the cache. """
-        follower_cache = open("caches/CacheFollowers.txt", "r")
-        for line in follower_cache:
-            if user in line:
-                follower_cache.close()
-                return True
-        follower_cache.close()
+    def __init__(self, cacheName):
+        super().__init__()
+
+        self.cacheName = cacheName
+
+    def in_cache(self, user):
+        """ Returns if user is in the cache, and if so, the time in secs """
+        with open(self.cacheName) as cache:
+            cachedata = json.load(cache)
+        if user in cachedata.keys():
+            return True
```

Comment:
A very slick way to take care of the return functionality would be to get rid of the `if`, and use:
```python
return user in cachedata
```